hail_scripts/v02/utils/computed_fields/vep.py

```diff
@@ -78,7 +78,7 @@ def get_expr_for_vep_gene_ids_set(vep_transcript_consequences_root, only_coding_
 
 def get_expr_for_vep_protein_domains_set(vep_transcript_consequences_root):
     return hl.set(
-        vep_transcript_consequences_root.flatmap(lambda c: c.domains.map(lambda domain: domain.db + ":" + domain.name))
```

Comment:
Ran into an error, looks like https://github.com/macarthur-lab/hail-elasticsearch-pipelines/blob/6cdd8a66607c02de6fc860290e26fcff190fcf0d/hail_scripts/v02/utils/computed_fields/vep.py#L145 converted it to a string.

Reply:
What name do you propose? And would that mean updating the v01 equivalent https://github.com/macarthur-lab/hail-elasticsearch-pipelines/blob/94597bf14272e6ee022edb49c003c21bbc0208ec/hail_scripts/v01/utils/computed_fields.py#L327, as well as the default `vep_transcript_consequences_root` to sorted instead of original?

Reply:
> What name do you propose?

`vep_sorted_transcript_consequences_root` instead of `vep_transcript_consequences_root`, similar to [get_expr_for_vep_gene_id_to_consequence_map](https://github.com/macarthur-lab/hail-elasticsearch-pipelines/blob/6bbc5c6b0a00ab83ed3415d3a6a0a5e5bc57f319/hail_scripts/v02/utils/computed_fields/vep.py#L252) and [get_expr_for_worst_transcript_consequence_annotations_struct](https://github.com/macarthur-lab/hail-elasticsearch-pipelines/blob/6bbc5c6b0a00ab83ed3415d3a6a0a5e5bc57f319/hail_scripts/v02/utils/computed_fields/vep.py#L283).

I think the Hail 0.1 version can be changed separately.

Reply:
K how about I revert the change and keep the method unchanged, and create a new one with that name for this?

Reply:
Ok. It looks like this change would break load_clinvar_to_es.py, since that calls it with the original VEP consequences.

https://github.com/macarthur-lab/hail-elasticsearch-pipelines/blob/6bbc5c6b0a00ab83ed3415d3a6a0a5e5bc57f319/hail_scripts/v02/load_clinvar_to_es.py#L73

Reply:
So create two methods good?

Reply:
I guess it's also possible to do a check on the type before, I just hate making changes without being able to test against older ones haha.